PowerEditor/installer/themes/Solarized.xml

```diff
@@ -717,12 +1043,37 @@ Installation:
             <WordsStyle name="STRING2" styleID="7" fgColor="2AA198" bgColor="002B36" fontName="" fontStyle="0" fontSize="" />
             <WordsStyle name="OPERATOR" styleID="8" fgColor="93A1A1" bgColor="002B36" fontName="" fontStyle="0" fontSize="" />
             <WordsStyle name="INFIX" styleID="10" fgColor="CB4B16" bgColor="002B36" fontName="" fontStyle="0" fontSize="" />
-            <WordsStyle name="IDENTIFIER" styleID="9" fgColor="B58900" bgColor="002B36" fontName="" fontStyle="0" fontSize="" />
-            <WordsStyle name="INFIXEOL" styleID="11" fgColor="2AA198" bgColor="002B36" fontName="" fontStyle="0" fontSize="" />
-            <WordsStyle name="BACKTICKS" styleID="12" fgColor="B58900" bgColor="002B36" fontName="" fontStyle="0" fontSize="" />
-            <WordsStyle name="RAWSTRING" styleID="13" fgColor="2AA198" bgColor="002B36" fontName="" fontStyle="0" fontSize="" />
-            <WordsStyle name="RAWSTRING2" styleID="14" fgColor="2AA198" bgColor="002B36" fontName="" fontStyle="0" fontSize="" />
-            <WordsStyle name="ESCAPESEQUENCE" styleID="15" fgColor="0000FF" bgColor="002B36" fontName="" fontStyle="0" fontSize="" />

```

Comment:
These entries are added in https://github.com/notepad-plus-plus/notepad-plus-plus/commit/3e9413d6152db40e1cf31ecc8f2dd057451c998f (yesterday) and should not removed.

Reply:
Yeah I noticed those just then, I have put them back.

My apologies, this was unintentional.